AK/Optional.h

```diff
@@ -549,4 +549,5 @@ requires(IsLvalueReference<T>) class [[nodiscard]] Optional<T> {
 
 #if USING_AK_GLOBALLY
 using AK::Optional;
+using AK::OptionalNone;
```

Comment:
We could go with what `::std::optional` does and call it `nullopt` for extra "wait what is that" points 🤕 .

But as-is, changing the name would break Jakt :)

Reply:
Something like `Optional::None` or `Optional::Empty` would be nicer, but idk how Jakt relies on this, and mostly I just care about a clear and concise way of expressing an empty optional.

Reply:
Sounds like we'll need a follow-up PR then. 👍